futures-macro-async/src/lib.rs

```diff
@@ -573,19 +573,19 @@ if_nightly! {
         let mut spans = Tokens::new();
         tokens.to_tokens(&mut spans);
         let good_tokens = proc_macro2::TokenStream::from(spans).into_iter().collect::<Vec<_>>();
-        let first_span = good_tokens.first().map(|t| t.span).unwrap_or(Span::def_site());
-        let last_span = good_tokens.last().map(|t| t.span).unwrap_or(first_span);
+        let first_span = good_tokens.first().map(|t| t.span()).unwrap_or(Span::call_site());
```

Comment:
Should this really be `call_site` or better to make `first_last` function to return `proc_macro::Span`?